daemon/cluster/helpers.go

```diff
@@ -15,7 +14,7 @@ func getSwarm(ctx context.Context, c swarmapi.ControlClient) (*swarmapi.Cluster,
 	}
 
 	if len(rl.Clusters) == 0 {
-		return nil, errors.NewRequestNotFoundError(errNoSwarm)
+		return nil, notFoundError{errNoSwarm}
```

Comment:
I'd prefer to define `errNoSwarm` with the `notFoundError` wrapper. That will make it harder to return an untyped error by mistake.